pkg/pool-manager/virtualmachine_pool.go

```diff
@@ -259,6 +261,9 @@ func (p *PoolManager) initMacMapFromCluster(parentLogger logr.Logger) error {
 
 		if iface.MacAddress != "" {
 			if err := p.allocateRequestedVirtualMachineInterfaceMac(vmFullName, iface, parentLogger); err != nil {
+				if strings.Contains(err.Error(), "failed to allocate requested mac address") {
+					gauges.DuplicateMacGauge.Inc()
```

Comment:
question. if we have a duplicate mac in vms vm1,vm2, then won't KMP count 2 duplicates? (on on vm1 with vm2 and the other vm2 with vm1)

Reply:
The first one will enter the table without conflict, the 2nd won't enter and its the only one that will be counted